File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/Usuario.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DDS_Restaurant_Solution.Models
{
    public class Usuario
    {
        [Key]
        public int idUsuario { get; set; }

        [MaxLength(20)]
        public string user { get; set; }

        [MaxLength(40)]
        public string password { get; set; }

        public bool estado { get; set; }

        public DateTime lastLog { get; set; }

        public string numIdentidad { get; set; }
        [ForeignKey("numIdentidad")]
        public Empleado Empeado { get; set; }

        public int idTipoUsuario { get; set; }
        [ForeignKey("idTipoUsuario")]
        public TipoUsuario TipoUsuario { get; set; }
    }
}

File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/Usuario.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class Usuario
    <Key>
    Public Property idUsuario As Integer
    <MaxLength(20)>
    Public Property user As String
    <MaxLength(40)>
    Public Property password As String
    Public Property estado As Boolean
    Public Property lastLog As DateTime
    Public Property numIdentidad As String
    <ForeignKey("numIdentidad")>
    Public Property Empeado As Empleado
    Public Property idTipoUsuario As Integer
    <ForeignKey("idTipoUsuario")>
    Public Property TipoUsuario As TipoUsuario
End Class
